README.md

```diff
@@ -44,6 +46,39 @@ app.get('/', (req, res) => {
 })
 ```
 
+### Using resolveMode `PROMISE` with async handler
+
+```js
+// lambda.js
+'use strict'
+const awsServerlessExpress = require('aws-serverless-express')
+const app = require('./app')
+const server = awsServerlessExpress.createServer(app)
+
+// note that callback the promise is always resolved
+// i.e. errors still lead to a resolved promise
+// The resolved promise has a a statusCode property that can be checked for error codes
+
+exports.handler = async (event, context) => { return awsServerlessExpress.proxy(server, event, context, 'PROMISE') }
```

Comment:
the .proxy call should be suffixed with .promise I believe.

Reply:
If your handler is't doing any more async work why would someone use this feature?